persistence-common/src/main/java/io/stargate/db/datastore/common/AbstractCassandraSchemaConverter.java

```diff
@@ -146,12 +149,14 @@ private Table convertTable(String keyspaceName, T table, Iterable<V> views) {
     Stream<Index> secondaryIndexes = convertSecondaryIndexes(keyspaceName, table, columns);
     Stream<Index> materializedViews = convertMVIndexes(keyspaceName, table, views);
     String comment = comment(table);
+    int ttl = ttl(table);
```

Comment:
Good point. I went ahead and merged for now; we can figure out what to do wrt access to Views (as distinct from Tables). I guess it could be solved either at persistence or API/service layer. Or, specifically to suppress I guess API/service should that.